README.rst

```diff
@@ -143,7 +143,7 @@ Partition options:
 			* ``R``, ``raid``: Linux RAID (a19d880f-05fc-4d3b-a006-743f0f84911e)
 			* ``V``, ``lvm``: Linux LVM (e6d6d379-f507-44c2-a23c-238f2a3df928)
 			* ``F``, ``fat32``: FAT32 / Basic Data Partition (ebd0a0a2-b9e5-4433-87c0-68b6b72699c7)
-			* ``B``, ``barebox-state``: Barebox State (4778ed65-bf42-45fa-9c5b-287a1dc4aab1)
```

Comment:
Yes, this might become ambiguous with this PR, but I would keep it in place for backwards compatibility given that it made it into the v17 release.

Reply:
Given that this is a relatively new feature, I think the fallout of removing this again should be fairly manageable?

Reply:
I think, removing it is ok, but add a comment here in the documentation that it happened. So if someone gets an error and checks the docs, they will know why.

Reply:
Ok, I added a reference to the old shortcut.